src/main/java/edu/ucdavis/dss/ipa/entities/Schedule.java

```diff
@@ -244,4 +233,31 @@ public void toggleInstructorSupportCallReview(String term) {
 
 		this.setInstructorSupportCallReviewOpen(newTermsBlob);
 	}
+
+	/**
+	 * Will generate a list of terms from the instructorSupportCallReview termsBlob
+	 * Example: '1010000001' => ['01', '03', '10']
+	 * @return
+     */
+	@Transient
+	@JsonIgnore
+	public List<String> getInstructorSupportCallReviewAsTerms() {
+		List<String> terms = new ArrayList<>();
+
+		String termBlob = this.getInstructorSupportCallReviewOpen();
```

Comment:
Ensure termBlob exists, unless this condition is not possible due to DB constraints.